lib/FieldMapper/ContentFieldMapper/BlockDocumentsBaseContentFields.php

```diff
@@ -42,22 +42,30 @@ class BlockDocumentsBaseContentFields extends ContentFieldMapper
      */
     protected $sectionHandler;
 
+    /**
+     * @var string
```

Comment:
Maybe something like this (here and on the others):
```suggestion
     * @var string Pattern with special characters that should be stripped from field value due to Solr not accepting them.
```

Reply:
'regexp pattern' ?